source/client-side-encryption/tests/legacy/fle2v2-CreateCollection.yml

```diff
@@ -17,10 +16,7 @@ tests:
         kmsProviders:
           aws: {} # Credentials filled in from environment.
         encryptedFieldsMap:
-          default.encryptedCollection: &encrypted_fields0 {
-                "escCollection": "enxcol_.encryptedCollection.esc",
-                "eccCollection": "enxcol_.encryptedCollection.ecc",
-                "ecocCollection": "enxcol_.encryptedCollection.ecoc",
+          default.encryptedCollection: &encrypted_fields {
```

Comment:
Likewise, no reason to use `{` here. Consider the more idiomatic YAML syntax I suggested above:

```
default.encryptedCollection: &encrypted_fields
  fields:
    - path: "firstName"
      bsonType: "string"
      keyId: { $binary: { base64: "AAAAAAAAAAAAAAAAAAAAAA==", subType: "04" }}
```